NachoSpigot-Server/src/main/java/net/minecraft/server/MinecraftServer.java

```diff
@@ -571,6 +576,27 @@ public void run() {
                 //long lastTick = System.nanoTime(), catchupTime = 0, curTime, wait, tickSection = lastTick;
                 long start = System.nanoTime(), lastTick = start - TICK_TIME, catchupTime = 0, curTime, wait, tickSection = start;
                 // PaperSpigot end
+                
+             // Enable thread affinity
+        		if (NachoConfig.threadAffinity)
+        		{
+        			System.out.println(" ");
+        			System.out.println("Enabling Thread Affinity...");
+        			lock = AffinityLock.acquireLock();
+        			if (lock.cpuId() != -1)
+        			{
+        				System.out.println("CPU " + lock.cpuId() + " locked for server usage.");
+        				System.out.println("This will boost the server's performance, but will use more cpu.");
+        				System.out.println("This is most effective on linux with JNA installed.");
+        				System.out.println("See https://github.com/OpenHFT/Java-Thread-Affinity");
+        				System.out.println(" ");
+        			} else
+        			{
+        				System.out.println("An error occured whilst enabling thread affinity!");
+        				System.out.println(" ");
+        			}
+
+        		}
```

Comment:
```suggestion
                // Enable thread affinity
                if (NachoConfig.threadAffinity) {
                    System.out.println(" ");
                    System.out.println("Enabling Thread Affinity...");
                    lock = AffinityLock.acquireLock();
                    if (lock.cpuId() != -1) {
                        System.out.println("CPU " + lock.cpuId() + " locked for server usage.");
                        System.out.println("This will boost the server's performance, but will use more cpu.");
                        System.out.println("This is most effective on linux with JNA installed.");
                        System.out.println("See https://github.com/OpenHFT/Java-Thread-Affinity");
                        System.out.println(" ");
                    } else {
                        System.out.println("An error occured whilst enabling thread affinity!");
                        System.out.println(" ");
                    }
                }
```
better formatting in relation to the other code and i replaced the tab-stops

Reply:
Yeah sorry that is how I'm used to doing my brackets. I believe that the info on JNA should stay there but ig some of the other stuff can be shaved off.

Reply:
yeah its also a mess for me :D tab seems fine local but GH dont likes them and format it different

Reply:
Nice to know :)